blockchyp/blockchyp_client.py

```diff
@@ -101,49 +108,85 @@ def is_terminal_routed(self, request):
     def __gateway_get(self, path):
         url = self.gateway_host + "/api" + path
         gateway_config = self.__get_gateway_config()
-        return requests.get(url, gateway_config)
+        reply = requests.get(url, headers=gateway_config)
+
```

Comment:
add conditional logic here to branch between the test gateway and production gateway based on the test flag